pkg/agent/core/ngt/service/ngt.go

```diff
@@ -128,6 +130,8 @@ type ngt struct {
 
 	idelay time.Duration // initial delay duration
 	dcd    bool          // disable commit daemon
+
+	kvsdbConcurrency int // kvsdb concurrency
```

Comment:
I could not understand what is the `kvsdbConcurrency` and what does it work for.
could you please ELI5?

Reply:
kvsdb uses a HashTable in Vald when processing all cases, etc., so it is possible to search in parallel in each Hash space (512 bits).
It is possible to search up to 512 threads, but generating many goroutines is costly, so it is made so that the degree of concurrency can be limited from the configuration.